src/main/converter/opds.ts

```diff
@@ -49,14 +49,16 @@ export class OpdsFeedViewConverter {
     public convertOpdsLinkToView(link: OPDSLink, url: string): OpdsLinkView {
         // Title could be defined on multiple lines
         // Only keep the first one
-        let title = link.Title;
-        const titleParts = title.split("\n");
-        title = titleParts[0];
+        const titleParts = link.Title.split("\n").filter((text) => text);
+        const title = titleParts[0].trim();
+        const subtitle = titleParts[1] && titleParts[1].trim();
 
+        // FIXME : Does publicationCount works? I never see it in react
         return  {
             title,
+            subtitle,
             url: urlPathResolve(url, link.Href),
-            publicationCount: (link.Children) ? link.Children.length : null,
+            publicationCount: link.Children && link.Children.length,
```

Comment:
`link.Children` is not used in the OPDS model. `publicationCount` is always zero.

OPDS feeds can provide the Atom "Thread Count" (`ThrCount`) number:
https://readium2.herokuapp.com/opds-v1-v2-convert/https%3A%2F%2Fcatalog.feedbooks.com%2Fcategory%2FFBFIC000000%2Fsub.atom

...but as you can see, this is only available in the parsed XML (left column, OPDS1), there is no equivalent in the OPDS2 JSON conversion (right column).

Reply:
I have asked the question: https://github.com/opds-community/drafts/issues/30

Reply:
fix